src/routes/news/components/LocalNewsList.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import * as React from 'react'
+import { isEmpty } from 'lodash/lang'
+import styled from 'styled-components'
+import { LocalNewsModel } from '@integreat-app/integreat-api-client'
+
+const NoItemsMessage = styled.div`
+  padding-top: 25px;
+  text-align: center;
+`
+
+const StyledList = styled.div`
+  position: relative;
+  padding-top: 1px;
+  background: linear-gradient(to left, rgba(168, 168, 168, 0.2), #bebebe 51%, rgba(168, 168, 168, 0.2));
```

Comment:
Is it possible to use hex colors instead of rgba here?

Edit: https://stackoverflow.com/questions/15852122/hex-transparency-in-colors

Reply:
Also, why 51% instead of just 50%?

Reply:
ASFAIK rgba is more supported than `8 character hex notation` so I prefer to use it. 
As for the 51% I copied the style from the design but I probably should be 50% I'll change it.

Reply:
@klinzo Is there anything still relevant here?

Reply:
I guess not, nope.